docs/app-integration-development/20-Scoped-OAuth.md

```diff
@@ -12,20 +12,20 @@ tags: [app-integration-development]
 > need support.
 
 ## Register an App
-Extended OAuth Clients allow your application to act on a PagerDuty Account as a PagerDuty App. The access your application has to the PagerDuty Account is controlled by the scopes it is granted. Before you start building, you first need to register a PagerDuty App with an Extended OAuth Client. This is done via the Developer Mode UI in your PagerDuty Account.
+Scoped OAuth clients allow your application to act on a PagerDuty Account as a PagerDuty App. Your application's access to the PagerDuty Account is controlled by the scopes it is granted. Before you start building, you first need to [register a PagerDuty App](https://developer.pagerduty.com/docs/ZG9jOjExMDI5NTY5-register-an-app), then [add Scoped OAuth functionality](https://developer.pagerduty.com/docs/b25fd1b8acb1b-app-functionality). This is done via the Developer Mode UI in your PagerDuty Account.
```

Comment:
- here, should we use markdown linking like we do here: https://github.com/PagerDuty/developer-docs/blob/main/docs/webhooks/01-Overview.md#v1-webhooks-reached-end-of-support I know we use this method of linking often but these slugs can change and I wonder if we can end up linking to out of date documentation this way?
- Just a comment, but we are going to need to revisit the wording around "Functionality" since we're going to do away with that wording in the new version of the developer console